lesson08/script.js

```diff
@@ -35,9 +34,11 @@ function getList(){
 });
 }
 
-getList().then((value) =>{
+createLoading();
+
+SetDataToLists().then((value) =>{
```

Comment:
The "CreateLoading ();" function shouldn't be included in "SetDataToLists()" function, so I put here.

Reply:
```suggestion
function SetDataToLists(){
　createLoading();
    return new Promise((resolve,reject) => {
```
nits
It could be good here too.
I thought it would be easier to see it if I put it just before the "PROMISE" since it is used when activating the "PROMISE".
I thought it would be easier to see it if I put it just before that.

Reply:
I agree with you!
Considering "PROMISE", I think its position is certainly easy to understand.
I'll fix it again!